pydoc/developer/BUILDING_CHAPEL.md

```diff
@@ -0,0 +1,20 @@
+.. _building_chapel:
```

Comment:
Labels like this only work properly in `rst`. In markdown, they are just seen as plain text. If you want the labels to function, you can update to 
<pre><code>
```{eval-rst}  
.. _building_chapel:
```
</code></pre>

I am not sure these labels are needed though because I do not see them being used anywhere else. I would recommend just removing them.